config/schemas/config.yaml

```
@@ -14,6 +14,220 @@ $defs:
     thing:
       $ref: "#/$defs/thing"
     ```
+  affinity:
+    $comment: Real k8s resources might have existing schemas that could be reused as-is
+    additionalProperties:
+      type: object
+    default:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: node-role.kubernetes.io/control-plane
+                  operator: Exists
+    title: Affinity
+    type: object
+  commonResource:
```

```suggestion
  component:
```

Feels like a less conflicted name vs resource which is well used on it's own.

---

Done, tho the other sibling PRs will need rebasing later